### ISSUE TITLE
Make endosymbiosis target receive a population benefit

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#5126]
to help it stay around while being endosymsiosised

requires some change to auto-evo (something like bonus energy based on the energy gained by the species doing the endosymbiosing) or an external effect per endosymbiosis event giving some population increase.

**Comment by ZzZombo** [comment]
I don't think this works as intended in the current release. The species I targeted for the process went extinct in all patches after two out of the three required engulfs.

**Comment by Accidental-Explorer** [comment]
The PR that marked it as closed does provide a separate energy/population source for the endosymbionts, which _reduced_ the number of extinctions, but there is still some type of flaw in the core auto-evo system that can cause them to go extinct through migration or species splitting mechanics (I think).

That's kind of a different issue though, so I don't think this one should be re-opened.

**Comment by hhyyrylainen** [comment]
Yeah, they receive a *major* benefit now thanks to an exclusive miche just available for that species. So the condition set out by this issue is fulfilled. But further tweaking is likely needed, but it has to go through a different mechanism as the population benefit is already added.
